packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -121,33 +124,63 @@ class ExpansionPanelList extends StatelessWidget {
   /// The duration of the expansion animation.
   final Duration animationDuration;
 
+  /// Whether the Expansion Panel List can have multiple panels open
```

Comment:
looks like the other docs don't capitalize "Expansion Panel List". Please always make sure to maintain local style.